ethcore/src/test_helpers/evm_test_client.rs

```diff
@@ -286,12 +286,13 @@ impl<'a> EvmTestClient<'a> {
 		}).ok();
 		// Touching also means that we should remove the account if it's within eip161
 		// conditions.
-		self.state.kill_garbage(
-			&vec![env_info.author].into_iter().collect(),
-			schedule.kill_empty,
-			&None,
-			false
-		).ok();
+		if schedule.kill_empty {
+			self.state.kill_garbage(
+				&vec![env_info.author].into_iter().collect(),
```

Comment:
```suggestion
				std::iter::once(env_info.author).collect(),
```

Reply:
Enlighten me, why is this better?

Reply:
I guess it does only one allocation (for `HashSet`) instead of 2 (+ 1 for a `Vec`).

Reply:
Yes, that's my understanding too but I saw that this is just used for tests, so feel free to ignore it :)

Reply:
Ok, I'll keep it as-is then, for readability.